beta/project-griffon/troubleshoot-project-griffon.md

```diff
@@ -55,7 +55,7 @@ Please upgrade the Adobe Analytics and Assurance SDK extensions to the latest ve
 | react-native-aepassurance v2.x.x | [react-native-acpcore](https://www.npmjs.com/package/@adobe/react-native-acpcore) | npm install @adobe/react-native-aepassurance@^2.0.0 <br/>npm install @adobe/react-native-acpcore |
 | react-native-aepassurance v3.x.x | [react-native-aepcore](https://www.npmjs.com/package/@adobe/react-native-aepcore) | npm install @adobe/react-native-aepassurance@^3.0.0 <br/>npm install @adobe/react-native-aepcore |
 
-In case you are using `react-native-acpcore` with Assurance and the React Native application fails to build with one of the following error messages:
+If you are using `react-native-acpcore` with AEPAssurance, the React Native application can fail to build with one of the following error messages:
```

Comment:
Should we say `may fail` instead of `can fail`?

Reply:
Updated based on @calebk1's recommendation, any thoughts on this?

Reply:
`may fail` is also fine! 

Just generally, I prefer saying `can`, since (at least to me), `may fail` implies that it will fail a more significant chunk of time, as opposed to `can fail`, which implies that there is a chance of failure, as opposed to there being a normalized chance of failure?

But either of those are fine - that's just my thoughts on usage of may vs can :P